Gopkg.toml

```diff
@@ -48,15 +48,6 @@ required = [
   name = "sigs.k8s.io/cluster-api"
   revision = "af298a2a480fb7143786cf4f9990e741855f08e9"
 
-# TODO: It's currently necessary to pin this to kubernetes-1.13.1, instead of
-# kubernetes-1.13.4, due to some transitive dependencies upstream.
-# See here:
-#  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/116#discussion_r262599059
-#  - https://github.com/kubernetes-sigs/controller-runtime/pull/349
-[[constraint]]
-  name = "k8s.io/client-go"
-  version = "kubernetes-1.13.1"
```

Comment:
Any reason for removing this constraint here?

Reply:
This was in response to Tariq's concern here https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/136#discussion_r265398524

Reply:
I am actually torn between pinning vs non-pinning, but understand the need to just relying on upstream dependencies rather than putting a constraint or an override here, what do you think?

Reply:
Constraining or overriding here could very well lead to conflicts with the controller-runtime and controller-tools dependencies.